packages/next/server/image-optimizer.ts

```diff
@@ -584,18 +586,6 @@ function getSupportedMimeType(options: string[], accept = ''): string {
   return accept.includes(mimeType) ? mimeType : ''
 }
 
-function getHash(items: (string | number | Buffer)[]) {
-  const hash = createHash('sha256')
-  for (let item of items) {
-    if (typeof item === 'number') hash.update(String(item))
-    else {
-      hash.update(item)
-    }
-  }
-  // See https://en.wikipedia.org/wiki/Base64#Filenames
-  return hash.digest('base64').replace(/\//g, '-')
-}
```

Comment:
This change is likely semver major because it will invalidate the existing cache for all images